apps/glusterfs/app_cluster.go

```diff
@@ -143,12 +143,13 @@ func (a *App) ClusterDelete(w http.ResponseWriter, r *http.Request) {
 		}
 
 		err = entry.Delete(tx)
-		if err == ErrConflict {
-			http.Error(w, err.Error(), http.StatusConflict)
-			return err
-		} else if err != nil {
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-			return err
+		if err != nil {
+			if err == ErrConflict {
+				http.Error(w, entry.ErrConflictFmt(), http.StatusConflict)
+			} else {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+			}
+			return logger.Err(err)
```

Comment:
This is cool, but like the #503 we should not log ERROR on conflict.  ERROR is more of a "Hey this is messed up, fix this, i don't know what to do!".  Also, although I didn't point this out in #503, most likely there is no need to log errors in app_XXX .